cloud/aws/templates/aws_oidc/app.tf

```diff
@@ -344,8 +344,8 @@ resource "aws_iam_role_policy_attachment" "civiform_ecs_task_execution_role_cust
   policy_arn = aws_iam_policy.civiform_ecs_task_execution_role_custom_policy[count.index].arn
 }
 
-resource "aws_ecs_task_definition" "td" {
-  family = "${local.name_prefix}-td"
+resource "aws_ecs_task_definition" "civiform_with_monitoring" {
```

Comment:
**thought:** I'm fine with either option but wondering if we should keep the same name for the existing instance (with a comment that this one has monitoring) and update the name for the new instance. Just to ensure it isn't confusing when entities see something new. And the other definition can be "td_without_monitoring" or "civiform_only" or some combo of those.

Reply:
I'd like to move away from inscrutable names like `td_`.

> Just to ensure it isn't confusing when entities see something new.

I don't know how much people are reading the terraform output when they run a deployment. That being said, the only deployer using this system who isn't already aware of this change is Terry, it'd be easy to let him know to expect this. The benefit of the clearer variable name is worth it imo.

Reply:
That makes sense to me! Would you mind just mentioning in the arkansas-deployment channel once this is submitted.

Reply:
@dkatzz will do!